stacks/java-quarkus/1.5.0/devfile.yaml

```
@@ -0,0 +1,70 @@
+schemaVersion: 2.2.0
+metadata:
+  name: java-quarkus
+  displayName: Quarkus Java
+  description: Java application using Quarkus and OpenJDK 17
+  icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
+  tags:
+    - Java
+    - Quarkus
+  projectType: Quarkus
+  language: Java
+  version: 1.4.0
```

You have the incorrect stack version in the devfile:

```suggestion
  version: 1.5.0
```